### PR TITLE
Room Default Configuration

### PR DESCRIPTION
**Pull request by supun19**
type list-multi support

Thanks for making a pull request to converse.js!

Before submitting your request, please make sure the following conditions are met:

- [X] Add a changelog entry for your change in `CHANGES.md`
- [X] When adding a configuration variable, please make sure to
      document it in `docs/source/configuration.rst`
- [X] Please add a test for your change. Tests can be run in the commandline
      with `make check` or you can run them in the browser by running `make serve`
      and then opening `http://localhost:8000/tests.html`.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Changelog
-
+### Room Default Configuration
+  support list-multi
```
Comment (jcbrand) [inline CHANGES.md]: Please put this under the `## 5.0.0 (Unreleased)` heading instead of making your own heading.

**Comment by jcbrand** [comment]
Looking good @supun19, thanks!

At the moment we're also including `dist/converse.js` and `src/headless/dist/converse-headless.js` so please generate those files and include them in your PR.

You can generate them with `make dist/converse.js` and `make src/headless/dist/converse-headless.js`, or by running `make watch`.

File: CHANGES.md (hunk 1 above)
Comment (supun19) [reply CHANGES.md]: thanks@jcbrand

File: CHANGES.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Changelog
-### Room Default Configuration
-  support list-multi
-## 5.0.0 (Unreleased)
 
+## 5.0.0 (Unreleased)
+- Room Default Configuration
+  support list-multi
```
Comment (jcbrand) [inline CHANGES.md]: Please make this a descriptive sentence on one line, not two.

**Comment by jcbrand** [comment]
Thanks, I've merged manually.

**Pull request closed without merging**